### DISCUSSION TITLE
I locked a file, but can't unlock it, any ideas?

### DISCUSSION THREAD

**Discussion by qubitz** [spoiledcat/git-for-unity#11] [Q&A] [answered]
Our team was continuing to play around with locking files and we we're excited to see the locks propagate to other computers. Simple I know, but we're easy to please (sometimes). However, in our testing, it didn't seem that we were able to release a lock, only force it.

![image](https://user-images.githubusercontent.com/15070724/206330596-ab656bd1-4fe0-4995-9105-85d3e42a9144.png)

You can see that the file was locked to by me, but my only option is to force unlock it. I doubt that time duration would have an intended effect; it's been a few weeks since I locked the file.

Has anyone had a similar experience? What fixed it for you?

**Comment by shana** [comment, marked as answer]
hey hey! You haven't done anything wrong, the problem is that the logic that matches a github lfs user to your git user relied on github-specific APIs, and all of that was removed when this project became Git for Unity (as opposed to GitHub for Unity).

That means that the system currently doesn't know that you are the one who locked it, so it only gives you the force unlock option. The result is the same, in the end, the lock will be released.

Once I fix this so a new UI is added to ask you to confirm which lfs username is yours, then this won't be as confusing.

**Reply by qubitz** [reply]
Ah! That makes a ton of sense. Thanks for the explanation and work you've put into this.
